modules/twinkleblock.js

```diff
@@ -163,6 +170,33 @@ Twinkle.block.callback.change_action = function twinkleblockCallbackChangeAction
 
 	partial.prop('disabled', !blockBox && !templateBox);
 
+	// Add current block parameters as default preset
+	if (Twinkle.block.currentBlockInfo) {
+		Twinkle.block.blockPresetsInfo.prior = Twinkle.block.currentBlockInfo;
+		var prior = {
+			label: 'Prior block',
+			// value not a valid template selection, chosen below by setting templateName
+			list: [{ label: 'Prior block settings', value: 'prior', selected: true }]
+		};
+		// Arrays of objects are annoying to check
+		if (!blockGroup.some(function(bg) {
+			return bg.label === prior.label;
+		})) {
+			blockGroup.push(prior);
+		}
```

Comment:
Could also just add the `prior` group to both right off the bat if there's a current block.  Probably a net efficiency gain if there's any toggling going on.